helm/minio/values.yaml

```diff
@@ -245,6 +245,59 @@ resources:
   requests:
     memory: 16Gi
 
+## List of policies to be created after minio install
+##
+## In addition to default policies [readonly|readwrite|writeonly|consoleAdmin|diagnostics]
+## you can define additional policies with custom supported actions and resources
+policies: []
+# - name: writeexamplepolicy
+#   statements:
+#     - resources: 
+#         - 'arn:aws:s3:::example2'
+#         - 'arn:aws:s3:::example3'
+#       actions:
+#         - "s3:AbortMultipartUpload"
+#         - "s3:GetObject"
+#         - "s3:DeleteObject"
+#         - "s3:PutObject"
+#         - "s3:ListMultipartUploadParts"
+#     - resources: 
```

Comment:
some of these resources need to have wildcard at the end to work can you update?